licenses_changes.txt

```diff
@@ -400,6 +401,7 @@ GPL-3.0+	GPL v3 or later
 GPL-3.0+	GPL-3+
 GPL-3.0+	GPL-3.0+
 GPL-3.0+	GPLv3+
+GPL-3.0-with-Qt-Company-Qt-exception-1.1	GPL-3.0-with-Qt-Company-Qt-exception-1.1
```

Comment:
hmm.. that actually seems like an error

Reply:
at least it's a very concreate license name :)

Reply:
phew - just gh webui swallowing spaces/tabs apparently